test/functional/setup/create_logins_azure.sql

```diff
@@ -0,0 +1,18 @@
+--for this script to work in Azure, use sqlcmd to connect to master database 
+IF NOT EXISTS (SELECT name FROM sys.sql_logins WHERE name = 'test_password')
+BEGIN
+    CREATE LOGIN test_password WITH PASSWORD='! ;4triou';
```

Comment:
Issue with this line: `test_password` might exist in the database with a different password. This will cause the tests fail. Same applies for `test_password2` and `test_password2`